pages/blog/posts/apidays-paris-2023-recap.md

```diff
@@ -0,0 +1,51 @@
+---
+
+Last week, we had the privilege of partnering with AsyncAPI and OpenAPI with the first API Specs and Standards booth at the API Days Paris Conference, which resulted in a fantastic experience where attendees had the chance to easily connect with experts in one single booth, and we had the opportunity of learning from the experience of OpenAPI and AsyncAPI conference tracks.
+
+This is the first time we have had a booth at a conference, but we are sure it won't be the last. We returned energized for the great conversations, inspired by the great work of the other standards, excited with the great content, and impressed by the great work done by the API Days Team.
+
+## About API Days Paris
+Apidays is the leading business and technology conference series in APIs, and the programmable economy and Paris edition is the biggest event of the series. 
+
+<div className='flex flex-wrap justify-center items-center gap-4 w-full'>
+  <iframe width="500" height="300" src="https://apiglobal.b-cdn.net/wp-content/uploads/2020/04/APIDAYS-PARIS-22-TRAILER-2.mp4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+### The Conference Theme
+In today's intricate digital landscape, APIs have emerged as powerful conduits for innovation, bridging the gaps and unlocking new realms of possibility. This year, API Days Paris revolves around four strong pillars: "Software, Sovereignty, Society, and Sustainability."
+
+* **Software**: Software & APIs are the pulsing heart of the digital revolution, from the frameworks that power global platforms to the microservices optimizing niche solutions, Serveless, Kubernetes, Kafka, and many other technologies.
+* **Sovereignty**: How should sovereignty be defined in the age of the cloud, decentralization, and ubiquitous connectivity, all while empowering individuals and organizations to chart their distinct digital paths?
+* **Society**: Let's understand the societal implications of rapid technological advances to ensure that we leave no member of society behind, fostering an equitable and harmonious future.
+* **Sustainability**: Let's explore the sustainable development, deployment, and management of software solutions to grasp the importance of eco-friendly digital infrastructures, ethical AI, and the long-term impact of our digital choices.
+
+## Connecting with JSON Schema and API practitioners
+It has been great having the chance to connect with JSON Schema practitioners in person. We received great feedback and created great connections to follow up with.
+
+<div className='flex flex-wrap justify-center items-center gap-4 w-full'>
+    <img className='max-w-400 px-20' src='/img/posts/2023/apidays-paris-2023-recap/booth-apidays.jpeg'/>
+</div>
+
+## Meaningful conversations with organizations
+At JSON Schema, we need sponsors to help us deliver our Roadmap, and this event enabled some great opportunities that hopefully will turn into future sponsorships.
```

Comment:
This is good. I would also like to mention that we are not JUST looking for sponsorship in terms of money, but in terms of "people resource". We want to connect with stakeholders / those invested in JSON Schema and its ecosystem. We want those stakeholders to be more directly part of our community, and be involved in driving our roadmap, like you say.

I don't know how you want to phrase that. I will leave that up to you.

Reply:
Great point. Thanks Ben. I improved this section to address this in my last push.